src/sourcekit-lsp/LanguageClientManager.ts

```diff
@@ -231,13 +232,14 @@ export class LanguageClientManager {
         const serverPathConfig = lspConfig.serverPath;
         const serverPath =
             serverPathConfig.length > 0 ? serverPathConfig : getSwiftExecutable("sourcekit-lsp");
+        const sdkArguments = swiftDriverSDKFlags(true);
```

Comment:
How come the sdk flag needs passed via `-Xswiftc` here?

Reply:
SourceKit-LSP doesn’t support SwiftPM-styled SDK flags, so we need to directly pass the flags to `swiftc`.